lib/src/react16_suggestors/react_dom_render_migrator.dart

```diff
@@ -91,8 +99,8 @@ class ReactDomRenderMigrator extends GeneralizingAstVisitor
         ')',
       );
     } else if (renderFirstArg.toSource().startsWith('ErrorBoundary') &&
-        overReactImport == null) {
-      addOverReactPatch(reactDomImport.offset);
+        (overReactImport == null)) {
```

Comment:
#nit unnecessary parenthesis
```suggestion
        overReactImport == null) {
```

Reply:
:P